### PR TITLE
Add support for 'Page Up' and 'Page Down'

### PR DESCRIPTION
**Pull request by robertbachmann**
Use curses keys 'PPAGE' and 'NPAGE' instead of
'PAGE_UP' and 'PAGE_DOWN'

### CODE REVIEW

File: src/charCodeMapping.py [hunk 1]
```diff
@@ -13,8 +13,6 @@
 CODE_TO_CHAR.update((value, name[4:]) for name, value in vars(curses).items()
                     if name.startswith('KEY_'))
 # special exceptions
-CODE_TO_CHAR[4] = 'PAGE_DOWN'
```
Comment (pcottle) [inline src/charCodeMapping.py]: do you know what keys these are? been a while since i wrote this

File: src/screenControl.py [hunk 2]
```diff
@@ -383,9 +383,9 @@ def processInput(self, key):
             self.toggleXMode()
         elif key == 'c':
             self.beginEnterCommand()
-        elif key == ' ' or key == 'PAGE_DOWN':
```
Comment (pcottle) [inline src/screenControl.py]: great, i wanted to make sure we kept the space and b keys

**Comment by pcottle** [comment]
AHhhhh I swear I googled this and never got it to work. thanks so much, especially for adding the test! :tada:

**Comment by pcottle** [comment]
i might whip up a release in a few days, since this is pretty nice to have

File: src/charCodeMapping.py (hunk 1 above)
Comment (robertbachmann) [inline src/charCodeMapping.py]: On Linux 4 seems to be CTRL+D
Don't know about 20.

**Comment by pcottle** [comment]
New release is live!
https://github.com/Homebrew/homebrew-core/pull/370

**Pull request merged**